stdlib/sys.pyi

```diff
@@ -57,9 +57,20 @@ if sys.version_info >= (3, 8):
     pycache_prefix: str | None
 ps1: object
 ps2: object
+
+# TextIO is used instead of more specific types for the standard streams,
+# since they can be monkeypatched at runtime. At startup, the objects
```

Comment:
I would say "are often" rather than "can be", since any variable _could_ be monkeypatched at runtime, but for these variables, it's a common idiom that's sort-of officially blessed. (If it weren't, there wouldn't be any need for `sys.__stdin__`, `sys.__stdout__` and `sys.__stderr__`.)

```suggestion
# since they are often monkeypatched at runtime. At startup, the objects
```